libraries/freertos_plus/standard/pkcs11/utest/CMakeLists.txt

```diff
@@ -1,93 +1,116 @@
-project ("freertos_plus pkcs11 cmock unit test")
-cmake_minimum_required (VERSION 3.13)
-set(3rdparty_dir "${AFR_ROOT_DIR}/libraries/3rdparty")
-set(standard_dir "${AFR_ROOT_DIR}/libraries/freertos_plus/standard")
-set(kernel_dir "${AFR_ROOT_DIR}/freertos_kernel")
-
-set(preprocess_commands "-I ${3rdparty_dir}/pkcs11/ -D'CK_PTR=*' -D'NULL_PTR=0' \
-                         -D'CK_DEFINE_FUNCTION( returnType, name )=returnType name'\ 
-                         -D'CK_DECLARE_FUNCTION( returnType, name )=returnType name'\ 
-                         -D'CK_DECLARE_FUNCTION_POINTER( returnType, name )=returnType( CK_PTR name )'\
-                         -D'CK_CALLBACK_FUNCTION( returnType, name )=returnType( CK_PTR name )'")
-
-separate_arguments(unix_flags UNIX_COMMAND "${preprocess_commands}")
-
-# Mock library
-list(APPEND mock_list
-    "${kernel_dir}/include/portable.h"
-    "${3rdparty_dir}/pkcs11/pkcs11.h"
-    )
-
-list(APPEND preprocessed_mock_list
-    "${3rdparty_dir}/pkcs11/pkcs11.h"
-    )
-
-
-create_mock_list(pkcs11_mock 
-    "${mock_list}"
-    "${CMAKE_SOURCE_DIR}/tools/cmock/project.yml"
-    )
-preprocess_mock_list(pkcs11_mock ${preprocessed_mock_list} "${unix_flags}")
-
-
-target_compile_definitions(pkcs11_mock PUBLIC
-        portHAS_STACK_OVERFLOW_CHECKING=1
-        portUSING_MPU_WRAPPERS=1
-        MPU_WRAPPERS_INCLUDED_FROM_API_FILE
+    project ("freertos_plus pkcs11 cmock unit test")
+    cmake_minimum_required (VERSION 3.13)
+    set(3rdparty_dir "${AFR_ROOT_DIR}/libraries/3rdparty")
```

Comment:
looks like we can remove these to align with the other mock suites

Reply:
oups, i forgot about this one
i will remove it